server/src/config-helper.ts

```diff
@@ -122,32 +130,25 @@ const BASE_CONFIG: CommonVoiceConfig = {
 }
 
 let injectedConfig: CommonVoiceConfig
-let loadedConfig: CommonVoiceConfig
 
 export function injectConfig(config: Partial<CommonVoiceConfig>) {
   injectedConfig = { ...BASE_CONFIG, ...config }
 }
 
 export function getConfig(): CommonVoiceConfig {
-  if (injectedConfig) {
-    return injectedConfig
-  }
-
-  if (loadedConfig) {
-    return loadedConfig
-  }
-
-  let fileConfig = null
-
   try {
-    const config_path = process.env.SERVER_CONFIG_PATH || './config.json'
-    fileConfig = JSON.parse(fs.readFileSync(config_path, 'utf-8'))
+    if (injectedConfig) {
+      console.log('Loading config: reading injectedConfig ...')
+      return injectedConfig
+    }
+
+    if (BASE_CONFIG) {
+      console.log('Loading config: reading BASE_CONFIG ...')
+      return BASE_CONFIG
+    }
```

Comment:
There is no need to check whether `BASE_CONFIG` exists, because it's always set either with default values or environment variables. So we can we can shorten it here to:
```suggestion
    console.log('Loading config: reading BASE_CONFIG ...')
    return BASE_CONFIG
```